docs/index.rst

```diff
@@ -5,66 +5,85 @@ Tahoe-LAFS
 .. Please view a nicely formatted version of this documentation at
    http://tahoe-lafs.readthedocs.io/en/latest/
 
-Contents:
+   Please see the notes under "Organizing Tahoe-LAFS documentation" in
+   docs/README.txt if you are editing this file.
+
+Tahoe-LAFS is a Free and Open decentralized storage system.
+It distributes your data across multiple servers.
+Even if some of the servers fail or are taken over by an attacker,
+the entire file store continues to function correctly,
+preserving your privacy and security.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
+   :caption: Getting Started with Tahoe-LAFS
 
    about-tahoe
+
    Installation/install-tahoe
    Installation/install-on-windows
    Installation/install-on-linux
    Installation/install-on-desert-island
```

Comment:
`install-tahoe` seems to be in conflict with the other three `install-on-*` documents. I find it confusing to offer two conflicting ways of installing something.

But really the conflict is in naming only: These three are called "install" but to me look more like "build" documentation.  

May be better suited with the developer docs? I'd move them like in fad99192080e0d578d70c65c676aab27234b8c35.